libcloudforensics/providers/aws/forensics.py

```diff
@@ -17,7 +17,7 @@
 
 from libcloudforensics.providers.aws.internal.common import UBUNTU_1804_FILTER
 from libcloudforensics.providers.aws.internal import account
-from libcloudforensics import logging_utils
+from libcloudforensics import logging_utils, errors
```

Comment:
Does our styleguide allow multiple import?

Reply:
I think we have this kind of multiple imports a bit everywhere. I opt to leave it as is for now and opened #243 to keep track.